src/allmydata/test/test_common_util.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module has been ported to Python 3.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from future.utils import PY2
+if PY2:
+    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
+
+import random
+import unittest
+
+from allmydata.test.common_util import flip_one_bit
+
+
+class TestFlipOneBit(unittest.TestCase):
+
+    def setUp(self):
+        random.seed(42)  # I tried using version=1 on PY3 to avoid the if below, to no avail.
+
+    def test_accepts_byte_string(self):
```

Comment:
This seems fragile to changes in Python's random algorithm. Perhaps a better approach is to assert that the result is one bit away from the origin, and not assert a specific value?

Given that this is testing test code, probably not worth spending time on, though.